students/nardielloam.html

```diff
@@ -0,0 +1,21 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="">
+    <title>Anthony Nardiello</title>
+</head>
+<body>
+    <h1>Example Student</h1>
+    <p>Hi!! My name is Anthony Nardiello, a 4th year Undergraduate Student at Montana State University, majoring with a Bachelor's of Science in Computer Science. My interests within Computer Science include
+	Machine Learning, Artificial Intelligence, Systems Architecture, and Software Engineering!! Some of my hobbies outside of my studies include reading, writing, cooking, tea, amateur photography, and travel (when we can more safely
+	Travel). Check below to see my Linkedin and Github for professional development, or follow me on Twitter for more personal insights into life and the world around us! </p>
+    <ul>
+      <li><a href="https://www.linkedin.com/in/anthony-nardiello-47a965139/">Check out my Linkedin</a></li>
+      <li><a href="https://github.com/nardielloam7451">Check out my Github</a></li>
+	  <li><a href="https://twitter.com/Futurist7451">Check here for my Twitter Ramblings</a></li>
```

Comment:
same line up the <li> tags

Reply:
Latest commit fixes this issue by lining them up!